### PR TITLE
Nborovenskiy/edxoldmng 224/create a management command to transfer certificate configs to credentials

### PR DESCRIPTION
**Pull request by NikolayBorovenskiy**
**Description:** This PR is going to implement a management command to transfer certificate configs to credentials.

**Youtrack:** https://youtrack.raccoongang.com/issue/EDXOLDMNG-224

**Testing instructions:**

Please call `python manage.py cms migrate_cert_config --help` to see the command capability.

**Reviewers:**
- [ ] @GlugovGrGlib 
- [x] @NiedielnitsevIvan 
- [ ] @UvgenGen 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation in source code updated
- [ ] All related Confluence documentation is updated (including deployment documentation)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: cms/djangoapps/contentstore/management/commands/migrate_cert_config.py [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+        then an user of this command will be notified by a message.
+        """
+        for course_key in course_keys:
+            if course := modulestore().get_course(course_key):
+                if course_modes := CourseMode.objects.filter(
+                    course_id=course_key, mode_slug__in=CourseMode.CERTIFICATE_RELEVANT_MODES
+                ):
+                    if certificates := CertificateManager.get_certificates(course):
+                        for certificate_data in certificates:
+                            for mode in course_modes:
```
Comment (NiedielnitsevIvan) [inline cms/djangoapps/contentstore/management/commands/migrate_cert_config.py:221]: I understand correctly that several identical certificates with different mods can be sent to the Credentials here?

Comment (NikolayBorovenskiy) [reply cms/djangoapps/contentstore/management/commands/migrate_cert_config.py:221]: Yes, each mode that can have a certificate will get a request to Cred.

File: cms/djangoapps/contentstore/management/commands/migrate_cert_config.py [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    def __init__(self, id_: int):
+        self.id = id_
+
+
+class FakedRequest:
+    def __init__(self, user_id: int):
+        self.user = FakedUser(user_id)
+
+
+class Command(BaseCommand):
```
[the remark this answers is not in the document]
Comment (NikolayBorovenskiy) [reply cms/djangoapps/contentstore/management/commands/migrate_cert_config.py:39]: done. get a docstign)

[the remark this answers is not in the document]
Comment (NiedielnitsevIvan) [reply cms/djangoapps/contentstore/management/commands/migrate_cert_config.py:39]: resolved

File: cms/djangoapps/contentstore/management/commands/migrate_cert_config.py (hunk 1 above)
Comment (NiedielnitsevIvan) [reply cms/djangoapps/contentstore/management/commands/migrate_cert_config.py:234]: okay, resolved

**Pull request merged**